packages/react-components/react-dialog/src/components/DialogSurface/useDialogSurfaceStyles.styles.ts

```diff
@@ -55,24 +61,62 @@ const useBackdropResetStyles = makeResetStyles({
   ...shorthands.inset('0px'),
   backgroundColor: 'rgba(0, 0, 0, 0.4)',
   position: 'fixed',
+  transitionDuration: tokens.durationGentle,
+  transitionTimingFunction: tokens.curveLinear,
+  transitionProperty: 'opacity',
+  willChange: 'opacity',
+  opacity: 0,
+});
+
+const useRootMotionStyles = makeStyles({
+  visible: {
+    boxShadow: tokens.shadow64,
+    opacity: 1,
+    transform: 'scale(1) translateZ(0)',
+  },
+
+  entering: {
+    transitionTimingFunction: tokens.curveDecelerateMid,
```

Comment:
Heads up: #29473

Reply:
I guess we can keep it using the token and then add a FIXME label in this case